hack/build-in-container.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
```

Comment:
This script is not called from anywhere. If we are supposed to call it manually, can you add it to Makefile, or document it somewhere?